src/components/DroughtThresholds.vue

```diff
@@ -110,24 +125,39 @@ img {
 }
 #title-container {
   position: fixed;
+  background-image: url("./dry_drought_soil_publicDomain.jpg");
+  background-size: cover;
 }
 #scroll-container {
   z-index: 200;
 }
 .text-container {
   z-index: 500;
   border-radius: 25px;
-  background-color: #333534;
+  background: #C33F00;
+  //mix-blend-mode: overlay;
   max-width: 400px;
   p{
     padding: 25px;
   }
 }
+.knockout-text {
+  font-size: 20vh;
+  font-weight: bold;
+  font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;
+  color: #e76254;
+  background: white;
+  mix-blend-mode: lighten;
+}
+
+.smaller {
+  font-size: 3.75em;
+}
 .hydro-chart {
   height: auto;
   margin-top: 10%;
   margin-left: 10%;
-  background-color: white;
+  background-color: #3F729C;
```

Comment:
The contrast on the white background is really high but then less so with the chart, which mutes the chart elements visually compared to the rest

Reply:
Overall I think the color needs some balance. The light orange at the end is hard to contrast on the blue as well. the red on white looks super clean. Maybe focusing on one of these aesthetics and applying to the title and the chart will be effective

Reply:
Agree, the colors still need work.